common/solar_system_initializers/hfe_initializer.txt

```diff
@@ -231,14 +448,51 @@ hfe_initializer_egypt_main = {
 		}
 		change_orbit = @base_moon_distance
 		moon = {
-			name = Thoth
-			class = pc_barren
-			entity = barren_planet_01_destroyed_entity
-			orbit_angle = {
-				min = 90
-				max = 270
-			}
+			name = "Thoth's Heresy"
```

Comment:
No, this is important

Reply:
I added this as part of the story in a more expansive backstory. Thoth's accident on that moon is the reason he is a heretic.

We don't know the Thoth Station's name is Thoth and it also recalls the pyramid. It doesn't reveal that the station's name is Thoth.

Reply:
No, its not reason he is heretic

Reply:
IT IS NOW! I expanded on the story, just as I have in countless other places and in the Localization.

Reply:
Your expanded history conflicts with things im plan to do.

Reply:
How many years have you planned this? I finally got something done and I am sure the players of this Mod will be interested in it.

Have you considered factoring my work in when you expand it? Perhaps Thoth committed multiple heresies?

Reply:
From my pow current ju ahets have no idea who is thoth and what he is done, they just have an alert from some ancient sensor system with a message like known criminal found.
This is why you can just convince them to go away, they dont really want to kill him, just do duty to the ancestors.
If he somehow destroyed the satellite, it would cause great destruction and great motivation in terms of killing him.
In addition, the moon was colonized as the political center of power of the government.
Once will be content about it

Reply:
Hmm. Interesting, I never thought of it that way! If that's your plan, I can respect it. I'll change the name today.